fabric-message-api-v1/src/main/java/net/fabricmc/fabric/api/message/v1/ServerMessageDecoratorEvent.java

```diff
@@ -62,7 +62,7 @@
  * <pre><code>
  * ServerMessageDecoratorEvent.EVENT.register(ServerMessageDecoratorEvent.STYLING_PHASE, (sender, message) -> {
  *     // Apply orange color to messages sent by server operators
- *     if (sender != null && sender.server.getPlayerManager().isOperator(sender.getGameProfile())) {
+ *     if (sender != null &amp;&amp; sender.server.getPlayerManager().isOperator(sender.getGameProfile())) {
```

Comment:
Might be better to escape using `{@code}` instead of HTML `<code>`? See `LootEvents`:
https://github.com/FabricMC/fabric/blob/a37366baa305d3cf9a30678a6c48fb983c8b6308/fabric-loot-api-v2/src/main/java/net/fabricmc/fabric/api/loot/v2/LootTableEvents.java#L67-L71